banzai_nres/pdf.py

```diff
@@ -37,7 +37,7 @@ def do_stage(self, image):
                                  np.squeeze(flux[primary_order, :]), color='blue')
         template_line, = pl.plot(template['wavelength'], template['flux'], color='red', linewidth=0.5)
         pl.xlim([5140., 5220.])
-        pl.ylim([0., np.max(flux[primary_order, :])])
+        pl.ylim([0., np.nanmin([np.nanmax(flux[primary_order, :]), 2.0])])
```

Comment:
I've noticed this choice of 2.0 in a few places. Not that it matters a lot, but what was the thinking with 2 here?

Reply:
The 2.0 is somewhat arbitrary, but the idea is that if there is a lot of noise in the spectra or normalization errors we want that vertical range to show up in the plots, but if there are tall cosmic rays we _don't_ want to show the full vertical range of those, so that the details of the actual spectrum can be seen. The 2.0 is an attempt to balance those factors.